conf/application.conf

```diff
@@ -184,6 +184,7 @@ baseUrl {
   direct-debit-update-email-frontend-local = "http://localhost:10801"
   gg = "http://localhost:9949/auth-login-stub/gg-sign-in"
   sign-out = "http://localhost:9949/auth-login-stub/session/logout"
+  contact-frontend = "http://localhost:9250"
```

Comment:
could we call this `contact-frontend-local` or something to indicate that we're not supposed to provide values for this in the app-configs?

Reply:
yeah sure thing